Assets/Mirage/Runtime/IServerObjectManager.cs

```diff
@@ -30,9 +30,7 @@ public interface IServerObjectManager
 
         void Spawn(GameObject obj, Guid assetId, INetworkPlayer owner = null);
 
-        void Destroy(GameObject obj);
-
-        void UnSpawn(GameObject obj);
+        void Destroy(GameObject obj, bool destroyServerObject);
```

Comment:
```suggestion
        void Destroy(GameObject obj, bool destroyServerObject = true);
```

use same default value as inside ServerObjectManager so we dont need to give arg if using interface